README.md

```diff
@@ -123,20 +180,28 @@ amioChat.messages.send({
 })
 ```
 
-### messages.sendText(text)
+### messages.sendText(text, metadata)
 Sends a text message. This is just a handy shortcut for `messages.send({type: 'text', payload: '...'}, metadata)`
 
 Parameters:
 - **text** - The content of the text message.
 - **metadata** - Optional. Add [metadata](https://docs.amio.io/v1.0/reference#messages-metadata) to the message. Metadata has to be an object and can carry whatever data needed to be sent along with the message.
 
 ### messages.sendImage(url, metadata)
-Sends an image message. This is just a handy shortcut for `messages.send({type: 'image', payload: '...'})`
+Sends an image message. This is just a handy shortcut for `messages.send({type: 'image', payload: '...'}, metadata)`
 
 Parameters:
 - **url** - The URL of the image.
 - **metadata** - Optional. Add [metadata](https://docs.amio.io/v1.0/reference#messages-metadata) to the message. Metadata has to be an object and can carry whatever data needed to be sent along with the message.
 
+### messages.sendQuickReply(text, quickReplyPayload, metadata)
```

Comment:
quick replies can be used even with all types of messages (structure, image, ...) - https://docs.amio.io/v1.0/reference#facebook-messenger-messages-quick-replies